internal/pkg/update/client.go

```diff
@@ -57,7 +57,7 @@ func NewClient(params *ClientParams) *client {
 		params.CheckInterval = 24 * time.Hour
 	}
 	if params.OS == "" {
-		params.OS = GetOs()
+		params.OS = runtime.GOOS
```

Comment:
what is this change, do you need to refresh from master

Reply:
Looks like a faulty merge/rebase.